mailgun/resource_mailgun_domain.go

```diff
@@ -109,6 +109,12 @@ func resourceMailgunDomain() *schema.Resource {
 					},
 				},
 			},
+			"dkim_key_size": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
ForeNew is true because any change to this value should recreate the resource

Reply:
100% agree.